modules/apps/transfer/types/msgs.go

```diff
@@ -113,6 +117,13 @@ func (msg MsgTransfer) ValidateBasic() error {
 		}
 	}
 
+	if msg.Forwarding.Unwind {
+		// When unwinding, we must have at must one token.
+		if len(msg.GetCoins()) > 1 {
+			return errorsmod.Wrap(ErrInvalidForwarding, "cannot unwind more that one tokens")
```

Comment:
```suggestion
			return errorsmod.Wrap(ErrInvalidForwarding, "cannot unwind more that one token")
```
Also, I'm thinking that instead of `ErrInvalidForwarding`, we could use `ErrInvalidCoins`, it's not the forwarding info that's bad, but the tokens that are bade based on the configured forwarding.

Reply:
Yeah that makes sense! Added a test as well